tests/ConsumerConfigurationTest.cc

```diff
@@ -20,12 +20,15 @@
 #include <lib/LogUtils.h>
 #include <pulsar/Client.h>
 
+#include <climits>
+
 #include "NoOpsCryptoKeyReader.h"
 
 DECLARE_LOG_OBJECT()
 
 #include "../lib/Future.h"
 #include "../lib/Utils.h"
+#include "pulsar/DeadLetterPolicyBuilder.h"
```

Comment:
```suggestion
#include <pulsar/DeadLetterPolicyBuilder.h>
```

Make include style consistent